assets/components/src/footer/index.js

```diff
@@ -14,78 +15,86 @@ import { ExternalLink } from '@wordpress/components';
 import { PatronsLogo } from '../';
 import './style.scss';
 
-const Footer = () => {
-	const componentsDemo = window && window.newspack_urls && window.newspack_urls.components_demo;
-	const setupWizard = window && window.newspack_urls && window.newspack_urls.setup_wizard;
-	const resetUrl = window && window.newspack_urls && window.newspack_urls.reset_url;
-	const resetWpcomUrl = window && window.newspack_urls && window.newspack_urls.reset_wpcom_url;
-	const pluginVersion = window && window.newspack_urls && window.newspack_urls.plugin_version;
-	const removeStarterContent =
-		window && window.newspack_urls && window.newspack_urls.remove_starter_content;
-	const footerElements = [
-		{
-			label: pluginVersion.label,
-			url: pluginVersion.url,
-		},
-		{
-			label: __( 'About', 'newspack' ),
-			url: 'https://newspack.pub/',
-			external: true,
-		},
-		{
-			label: __( 'Documentation', 'newspack' ),
-			url: 'https://newspack.pub/support/',
-			external: true,
-		},
-	];
-	if ( componentsDemo ) {
-		footerElements.push( {
-			label: __( 'Components Demo', 'newspack' ),
-			url: componentsDemo,
-		} );
-	}
-	if ( setupWizard ) {
-		footerElements.push( {
-			label: __( 'Setup Wizard', 'newspack' ),
-			url: setupWizard,
-		} );
-	}
-	if ( resetUrl ) {
-		footerElements.push( {
-			label: __( 'Reset Newspack', 'newspack' ),
-			url: resetUrl,
-		} );
-	}
-	if ( resetWpcomUrl ) {
-		footerElements.push( {
-			label: __( 'Reset WordPress.com Authentication', 'newspack' ),
-			url: resetWpcomUrl,
-		} );
-	}
-	if ( removeStarterContent ) {
-		footerElements.push( {
-			label: __( 'Remove Starter Content', 'newspack' ),
-			url: removeStarterContent,
-		} );
-	}
-	return (
-		<div className="newspack-footer">
-			<PatronsLogo />
-			<div className="newspack-footer__inner">
-				<ul>
-					{ footerElements.map( ( { url, label, external }, index ) => (
-						<li key={ index }>
-							{ external ? (
-								<ExternalLink href={ url }>{ label }</ExternalLink>
-							) : (
-								<a href={ url }>{ label }</a>
-							) }
-						</li>
-					) ) }
-				</ul>
+class Footer extends Component {
```

Comment:
React best practice as I understand it is to use functional components whenever possible. Since it seems you only converted to a class-based components to extract the `simple` prompt, couldn't we keep this as a functional component and get the prop from args? e.g. 

`
const Footer = ( { simple } ) => {
`

Reply:
I did this in order to have consistency with our other components but you're right, it's not needed 😊 
Fixed in https://github.com/Automattic/newspack-plugin/pull/885/commits/20fd6a98a041d10f4fe77f48082d6ac6ff5812ee

Reply:
Understood! Yeah, I think if we want to strive toward consistency across components we should refactor the class-based ones to be functional wherever we can, not the other way around. Thanks!